src/test/scala/com/amazon/deequ/analyzers/jdbc/JdbcAnalyzerTests.scala

```diff
@@ -177,11 +189,9 @@ class JdbcAnalyzerTests
         DoubleMetric(Entity.Mutlicolumn, "Uniqueness", "nonUnique,onlyUniqueWithOtherNonUnique",
```

Comment:
Can you remove or uncomment this comment?

Reply:
I removed it.